packages/actions/src/Concerns/CanImportRecords.php

```diff
@@ -334,18 +334,36 @@ protected function setUp(): void
                         $columns,
                     ));
 
-                    $example = array_map(
-                        fn (ImportColumn $column) => $column->getExample(),
+                    /*
+                    Сount the number of lines in the file.
+                    It should be equal to the largest number of examples in the columns.
+                    */
+                    $rowsCount = 0;
+                    array_map(
+                        function (ImportColumn $column) use (&$rowsCount) {
+                            $rowsCount = count($column->getExamples()) > $rowsCount ? count($column->getExamples()) : $rowsCount;
+                        },
                         $columns,
                     );
 
-                    if (array_filter(
-                        $example,
-                        fn ($value): bool => filled($value),
-                    )) {
-                        $csv->insertOne($example);
+                    /*
+                    Get an array of “examples” in which the number of elements
+                    is equal to the number of lines in the file.
+                    */
+                    $examples = [];
+                    foreach ($columns as $column) {
+                        $columnExamples = $column->getExamples();
+
+                        for($i = 0; $i < $rowsCount; $i++) {
+                            $examples[$i][] = $columnExamples[$i] ?? '';
+                        }
```

Comment:
Is it possible to refactor this to a simple `foreach()` through `array_values($column->getExamples())` and using the indexes from that array?

Reply:
I think not. Because one column may have fewer examples than another. We need to go through a `for`, check if the column contains fewer examples than the number of lines in the file, if so, add an empty value.